reactors/1D_packbed.ipynb

```diff
@@ -0,0 +1,518 @@
+    "$$\n",
+    "    \\beta_{\\rm g} = \\frac{\\phi_{\\rm{g}}^3 D_{\\rm p}^2}{72 \\tau_{\\rm{g}}(1 - \\phi_{\\rm{g}})^2},\n",
+    "$$\n",
+    "where $\\phi_{\\rm{g}}$, $\\tau_{\\rm{g}}$, and $D_{\\rm {p}}$ are the bed porosity, tortuosity, and particle diameter, respectively.\n",
+    "\n",
+    "The independent variable in these conservation equations is the position $z$ along the reactor length.  The dependent variables include total mass flux $\\rho u$, pressure $p$, temperature $T$, gas-phase mass fractions $Y_k$, and surfaces coverages $\\theta_k$. Gas-phase fluxes through the membrane are represented as $j_{k, {\\rm M}}$. Geometric parameters $A_{\\rm s}$, $P_{\\rm b}$, and $A_{\\rm b}$ represent the catalyst specific surface area (i.e., surface area per unit volume), reactor perimeter, and reactor cross-sectional flow area, respectively. Other parameters include bed porosity $\\phi_{\\rm g}$ and gas-phase species molecular weights $W_k$. The gas density $\\rho$ is evaluated using the equation of state (ideal Eos, RK or PR EoS).\n",
+    "\n",
+    "If a perm-selective membrane is present, then $j_{k_{\\rm M}}$ represents the gas-phase flux through the membrane and ${k_{\\rm M}}$ is the gas-phase species that permeates through the membrane. The Kronecker delta, $\\delta_{k, k_{\\rm M}} = 1$ for the membrane-permeable species and  $\\delta_{k, k_{\\rm M}} = 0$ otherwise. The membrane flux is calculated using Sievert's law as\n",
+    "$$\n",
+    "    j_{k_{\\rm M}}^{\\rm{Mem}} = \\frac{B_{k_{\\rm M}}}{t} \\left ( p_{k_{\\rm M}, {\\rm{mem}}}^\\alpha - p_{k_{\\rm {M,                 sweep}}}^\\alpha \\right ) W_{k_{\\rm M}}\n",
+    "$$\n",
+    "where $B_{k_{\\rm M}}$ is the membrane permeability, $t$ is the membrane thickness. $p_{k_{\\rm M}, {mem}}$ and $p_{k_{\\rm {M, sweep}}}$ represent perm-selective species partial pressures within the packed-bed and the exterior sweep channel. The present model takes the pressure exponent $\\alpha$ to be unity. The membrane flux for all other species ($ k \\neq k_{\\rm M}$) is zero.\n",
+    "\n",
+    "### Chemistry mechanism\n",
+    "This example uses a detailed 12-step elementary micro-kinetic reaction mechanism that describes ammonia formation and decomposition kinetics over the Ru/Ba-YSZ catalyst. The reaction mechanism is developed and validated using measured performance in a laboratory-scale packed-bed reactor[3]. This example also incorporates a Pd-based H$_2$ perm-selective membrane.\n",
+    "\n",
+    "\n",
+    "### Solver\n",
+    "\n",
+    "Above governing equations represent a complete solution for a steady-state packed-bed, catalytic membrane reactor model. The dependent variables are the mass-flux $\\rho u$, species mass-fractions $Y_k$, pressure $p$, temperature $T$, and surface coverages $Z_k$. The equation of state is used to obtain the mass density, $\\rho$.\n",
```

Comment:
This is not the same set of symbols for dependent variables as above. It's otherwise duplicated from above and can probably be removed.

Reply:
Corrected.